### ISSUE TITLE
Exception when switching devices in preview

### ISSUE THREAD

**Issue by billreed63** [maqetta/maqetta#3148]
FF/Win7

Create a mobile page

```
<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8"/>
<title>Untitled</title>
<script type="text/javascript" src="lib/dojo/dojo/dojo.js" data-dojo-config="'parseOnLoad':true,'packages':[{'name':'gridx','location':'../gridx'},{'name':'clipart','location':'../../clipart'},{'name':'maqettaSamples','location':'../../../samples'},{'name':'maqetta','location':'../../maqetta'},{'name':'shapes','location':'../../shapes'},{'name':'zazl','location':'../../zazl'},{'name':'widgets','location':'../../custom'}],'themeMap':[['Android','',['themes/iphone/iphone.css']],['BlackBerry','',['themes/blackberry/blackberry.css']],['iPad','',['themes/ipad/ipad.css']],['iPhone','',['themes/iphone/iphone.css']],['.*','',['themes/iphone/iphone.css']]],'mblThemeFiles':[], mblLoadCompatPattern: /\/project1\/themes\/.*\.css$/"></script>
<script type="text/javascript">
require([
  "dijit/dijit",
  "dojo/parser",
  "maqetta/space",
  "maqetta/AppStates",
  "dojox/mobile",
  "dojox/mobile/deviceTheme",
  "dojox/mobile/compat"
]);
</script>
<style>@import "themes/claro/claro.css";@import "app.css";
</style>
<script type="text/javascript" src="app.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no"></meta>
</head>
<body data-maq-device="iphone" data-maq-flow-layout="true" data-maq-ws="collapse" data-maq-appstates="{}" class="claro" id="myapp">
 <h1 label="Heading" data-dojo-type="dojox.mobile.Heading"></h1>
</body>
</html>

```

Preview,  in preview switch to android device
Note the following exception:

iframeSearch is not defined

Line 5568

**Comment by JonFerraiolo** [comment]
Wasn't able to reproduce on Mac/FF, so time to open up my Windows machine.

**Comment by billreed63** [comment]
@peller  This appears to be another zazl bug, if I turn of zazl it does away

**Comment by peller** [comment]
I think this one is fixed.  @billreed63 which build are you running?

**Comment by billreed63** [comment]
it is whatever is on the test server

**Comment by peller** [comment]
I believe it was fixed by e12a14aa049cde876fc2b1c7a32834c5fa80f9db.  The build on the test machines predates that.
